### PR TITLE
Fix prescribed drug frequency type adapter

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

**Review by harimohanraj89 — APPROVED** [review]
That looks correct to me. Are the unit tests that need to be updated?

**Comment by msasikanth** [comment]
> 
> 
> That looks correct to me. Are the unit tests that need to be updated?

I don't think we need to update any unit tests for this. But I want to wait until Monday before we merge this in to make sure if need to handle the data that has been entered in the SBX builds (I doubt we need to).

It won't effect the release, we will merge this is in first and then make a DEMO release on Monday.

**Pull request merged**
